eos/form-factors/parametric-kkvdz2022.cc

```diff
@@ -0,0 +1,185 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Stephan Kürten
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/form-factors/parametric-kkvdz2022.hh>
+#include <eos/maths/complex.hh>
+#include <eos/maths/power-of.hh>
+
+namespace eos
+{
+
+    static const constexpr double pi = M_PI;
```

Comment:
Please use the processes traits as recorded in ``mesonic-processes``: https://github.com/eos/eos/blob/ac94fe4d7153e9bac2e76629ef14a0b6fc7d260f/eos/form-factors/mesonic-processes.hh#L172-L190